packages/xmovie.yml

```diff
@@ -2,7 +2,7 @@ repo: jbusecke/xmovie
 section: animations
 description: A simple way of creating beautiful movies from xarray objects.
 site: # optional, default repo site
```

Comment:
I guess we shouldn't have these comments in here.  Just delete any empty lines I think.  Sorry for the bother!

Reply:
Ahh, thanks for the clarification. Some explanation in the comments would have helped a lot indeed.

Reply:
I think I fixed this so folks won't have this problem again....